mempool/mempool
# Fix "indexing in progress" message in ln dashboard

Fixes https://github.com/mempool/mempool/issues/2308

#### Testing

* Run the lightning backend
* Wait for for the info message `Updated latest network stats`
* Run the mysql query `truncate lightning_stats`
* Refresh the LN dashboard, make sure you are not being served cached response
* Confirm the message/layout is correct as seen below

**Desktop**

<img width="564" alt="Screen Shot 2022-08-18 at 2 57 29 PM" src="https://user-images.githubusercontent.com/9780671/185400389-da327727-d894-460e-83a8-e3f4942a5a95.png">

**Mobile**

<img width="364" alt="Screen Shot 2022-08-18 at 2 44 46 PM" src="https://user-images.githubusercontent.com/9780671/185399342-f22c14b3-1841-4c10-8ae2-0c8ae0b92453.png">
